compatibility/compatibility-vec11to12/src/test/resources/logback-test.xml

```diff
@@ -5,7 +5,7 @@
         </encoder>
     </appender>
 
-    <logger name="org.springframework" level="warn"/>
+    <logger level="warn"/>
```

Comment:
I'm not sure if that's a valid configuration / how to set a global log level.

We already defined that a few lines below with `root level="debug"`. I would simply remove this line.